src/daemon/absorber_test.cc

```diff
@@ -6,6 +6,39 @@
 namespace dist_clang {
 namespace daemon {
 
+namespace {
+
+// S1..4 types can be one of the following:
+// |String|, |ConstString|, |Literal| or |const char*|
```

Comment:
1. `const char*` is highly discouraged and shouldn't be mentioned.
2. `ConstString` is known as `Immutable`  right now.